### ISSUE TITLE
Can not persist changes

### ISSUE THREAD

**Issue by jsalatiel** [nextcloud/documentserver_community#46]
I can not save changes to  files. Usually only the FIRST save will persist.
All following changes will be missed.

Nextcloud 18.0.0 and Document Community Server 0.1.3

```
[documentserver_community] Error: OCA\DocumentServer\Document\DocumentConversionException: save_code: d.rb.ua,La,d.Me,e.Document)})}null!=d.fa&&(this.memory.va(Ai.fa),this.ra.Dk(function(){e.Epb(d,a,!0,c)}))};this.Epb=function(d,a,c,z){var h=0,Ga=d.fa.length-1;if(1==a&&(h=d.Selection.xa,Ga=d.Selection.ya,h>Ga)){var f=Ga;Ga=h;h=f}0>Ga&&(Ga=0);0>h&&(h=0);f=d.fa;for(var La=this;h<=Ga&&h<f.length;++h){var n=f[h];switch(n.ka){case $f:var r=n.Ym();ff!==r?it(r,n.Qv(),function(d,a,c,e){oh===d?a.ewa()?d=Ai.Qca:(d=Ai.Pya,c=!0):d=a.zpa()?Ai.Mfa:Ai.mza;La.ra.ta(d,function(){gB(La.ra,La.bq.Qja++,a,{UE:function(){e(c)}})})},
save: Uncaught TypeError: Cannot read property 'ua' of null
DoctRenderer:<result><error code="save" /></result>
 at <<closure>>

0. /var/www/nextcloud/apps/documentserver_community/lib/DocumentConverter.php line 123
   OCA\DocumentServer\Document\ConverterBinary->run("/tmp/oc_tmp_aRGR3S-.xml")
1. /var/www/nextcloud/apps/documentserver_community/lib/DocumentConverter.php line 78
   OCA\DocumentServer\DocumentConverter->runCommand(OCA\DocumentServ ... {})
2. /var/www/nextcloud/apps/documentserver_community/lib/Document/DocumentStore.php line 151
   OCA\DocumentServer\DocumentConverter->saveChanges("*** sensitive parameter replaced ***", ["78;AgAAADEA//8 ... "], "/nextcloud-data ... x", "docx")
3. /var/www/nextcloud/apps/documentserver_community/lib/LocalAppData.php line 100
   OCA\DocumentServer\Document\DocumentStore->OCA\DocumentServer\Document\{closure}("*** sensitive parameters replaced ***")
4. /var/www/nextcloud/apps/documentserver_community/lib/Document/DocumentStore.php line 164
   OCA\DocumentServer\LocalAppData->getReadWriteLocalPath(OC\Files\SimpleFS\SimpleFolder {}, Closure {})
5. /var/www/nextcloud/apps/documentserver_community/lib/Document/SaveHandler.php line 56
   OCA\DocumentServer\Document\DocumentStore->saveChanges(2060669791, [OCA\DocumentSer ... }])
6. /var/www/nextcloud/apps/documentserver_community/lib/BackgroundJob/Cleanup.php line 69
   OCA\DocumentServer\Document\SaveHandler->flushChanges(2060669791)
7. /var/www/nextcloud/lib/public/BackgroundJob/Job.php line 79
   OCA\DocumentServer\BackgroundJob\Cleanup->run(null)
8. /var/www/nextcloud/cron.php line 125
   OCP\BackgroundJob\Job->execute(OC\BackgroundJob\JobList {}, OC\Log {})

at 2020-01-28T16:50:04+00:00
```

**Comment by icewind1991** [comment]
0.1.4 includes some related fixes, can you see if the problem persists with the latest version

**Comment by jsalatiel** [comment]
Still broken, although no messages in log anymore. See the screenshots, please.

Create a  new file:

![image](https://user-images.githubusercontent.com/5522389/73377095-439eca00-429d-11ea-8389-20c95a95c5d0.png)

--
Edit the file:

![image](https://user-images.githubusercontent.com/5522389/73377126-50232280-429d-11ea-8b59-6d0841994ce0.png)

--
The file is there:

![image](https://user-images.githubusercontent.com/5522389/73377169-616c2f00-429d-11ea-8872-0dffeae2ddbc.png)

--

Try to open the file again. Completely corrupted!
![image](https://user-images.githubusercontent.com/5522389/73377194-6af59700-429d-11ea-8699-9cbc8be5dc5c.png)

**Comment by icewind1991** [comment]
Can you try to see if c14c3d2 fixes it

**Comment by jsalatiel** [comment]
It worked! thanks!

**Comment by jsalatiel** [comment]
I do not know if it is related to this , but there are parts of the document that i can not edit and onlyoffice thinks that my user is several users. See screenshot please. I can not delete the name pointed by the blue arrow and it appears to be 6 "me" users editing the document.
![image](https://user-images.githubusercontent.com/5522389/73383473-2a4f4b00-42a8-11ea-8567-fc0f5581860d.png)

**Comment by ostasevych** [comment]
> I do not know if it is related to this , but there are parts of the document that i can not edit and onlyoffice thinks that my user is several users. See screenshot please. I can not delete the name pointed by the blue arrow and it appears to be 6 "me" users editing the document.
> ![image](https://user-images.githubusercontent.com/5522389/73383473-2a4f4b00-42a8-11ea-8567-fc0f5581860d.png)

The same situation. If you create a file, add some text, close it, open again, this text is no more editable and mark with red dotted bracket at the left. The more interesting, that after several closing-opening and manual saving it started allow to edit the text. So it is locked either for certain time or with some other conditions, which are not described.

**Comment by icewind1991** [comment]
the "ghost sessions" will timeout after 120s

**Comment by icewind1991** [comment]
Fix is released with 0.1.5
